### PR TITLE
add read write values to continous-test

### PR DESCRIPTION
**Pull request by KyriosGN0**
#### Which issue(s) this PR fixes or relates to

allows setting a read and write URLs for continous-test, i recently deployed it in my org and since we dont use the gateway i had to hack it with extraArgs

#### Checklist

- [ ] Tests updated.
- [ ] Documentation added.
- [X] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

**Comment by KyriosGN0** [comment]
@56quarters, sorry for nagging, is something missing from my PR ?

**Comment by 56quarters** [comment]
> @56quarters, sorry for nagging, is something missing from my PR ?

Whoops, missed this. Sorry, no, there's nothing missing. I was just hoping someone with more helm knowledge would pick it up. I'll try to take a look next wee. Thanks!

**Review by 56quarters — APPROVED** [review]
LGTM, just one small fix. Thanks!

File: operations/helm/charts/mimir-distributed/CHANGELOG.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ Entries should include a reference to the Pull Request that introduced the chang
 * [BUGFIX] Add missing container security context to run `continuous-test` under the restricted security policy. #8653
 * [BUGFIX] Add `global.extraVolumeMounts` to the exporter container on memcached statefulsets #8787
 * [BUGFIX] Fix helm releases failing when `querier.kedaAutoscaling.predictiveScalingEnabled=true`. #8731
+* [ENHANCEMENT] Helm : allow setting a read and write urls to continous-test. #7674
```
[the remark this answers is not in the document]
Comment (KyriosGN0) [reply operations/helm/charts/mimir-distributed/CHANGELOG.md]: fixed

**Pull request merged**